src/localization/i18n.ts

```diff
@@ -0,0 +1,17 @@
+import i18n, {Callback, InitOptions} from 'i18next';
```

Comment:
rename file, usually file name the same as class/func name 
And 1 file for 1 class ...